README.md

```diff
@@ -1,10 +1,35 @@
+
+
+
 ![npm](https://img.shields.io/npm/v/@asyncapi/specs?style=for-the-badge) ![npm](https://img.shields.io/npm/dt/@asyncapi/specs?style=for-the-badge)
 
 > If you are currently using version 2, check out [migration guideline to version 3](./migrations/Migrate%20to%20version%203.md). You might be able to update it without any change.
 # AsyncAPI
 
 This is a mono repository, which provides all the JSON Schema documents for validating AsyncAPI documents.
 
+## Overview
+*  This repository contains [JSON Schema](https://json-schema.org/ for reference) files for all the versions of AsyncAPI specification.
```

Comment:
I always recommend to check the preview of the markdown and trying to click links, this one is a broken link

```suggestion
*  This repository contains [JSON Schema](https://json-schema.org) files for all the versions of AsyncAPI specification.
```

Reply:
Noted.Let me check on the suggestions. Thanks